docs/handbook/common-tasks.md

```diff
@@ -45,6 +45,87 @@ Print versions are also available.
 
 ## Basic system administration
 
+### Basic system information
+
+#### System processes
+
+```markdown
+prstat
+```
+
+This command provides a host of information on all processes running on the system.
+Some of the information provided is as follows:
+
+- percentage of CPU used by each process
+- amount of memory consumed by each process
+- unique id of each process (which can, for example, be used to stop the process)
+
+#### Disk usage
+
+```markdown
+df -h
+```
+
+Provides information on disk size, amount of space used and available free space
+for all attached storage devices. The `-h` option reports this information in
+human readable format.
+
+#### Largest files in a directory
+
+Go to the directory using the `cd` command and issue the following command:
+
+```markdown
+du | sort -n
+```
+
+This will list the size of each file in the current directory and all
+subdirectories, starting with the smallest up to the largest files.
+
+#### Who is logged on to the system
+
+```markdown
+listusers
+```
+
+#### List all software packages installed on the system
+
+```markdown
+pkginfo
+```
+
+### System shutdown, reboot, ...
+
+OpenIndiana defines a number of different system states known as run-levels. You
+can change from one system state to another by using the `shutdown` command and
+specify the run-level using the `i` option. You can always determine the
+run-level via `who -r`.
+
+You must be root or have root privilages (e.g., using `sudo`) to send the system
```

Comment:
About 'sudo', I'd prefer not to include a sudo nor a pfexec before each command.  What about if I'm logged in as root! I think we might devote a section to acquiring privilages.  There we could include a short description on pfexec, sudo and su.